### ISSUE TITLE
Synchronizing from Tomdroid wrecks syncronization from Ubuntu & Windows clients

### ISSUE THREAD

**Issue by obilodeau** [tomboy-notes/tomdroid#190]
Installed build 473.
Synchronized Tomdroid with notes via Ubuntu One
Modified a note.
Synchronized.
Now other clients fail with:

2012-09-08 09:29:56 [ERROR]: Synchronization failed with the following exception
: An error occurred while parsing EntityName. Line 4, position 81.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.Throw(String res, String arg)
   at System.Xml.XmlTextReaderImpl.Throw(String res)
   at System.Xml.XmlTextReaderImpl.ParseEntityName()
   at System.Xml.XmlTextReaderImpl.ParseEntityReference()
   at System.Xml.XmlTextReaderImpl.Read()
   at System.Xml.XmlTextReader.Read()
   at Tomboy.Sync.NoteUpdate..ctor(String xmlContent, String title, String uuid,
 Int32 latestRevision) in C:\Projects\tomboy-gnome\Tomboy\Synchronization\SyncMa
nager.cs:line 868
   at Tomboy.WebSync.WebSyncServer.GetNoteUpdatesSince(Int32 revision) in C:\Pro
jects\tomboy-gnome\Tomboy\Addins\WebSyncService\WebSyncServer.cs:line 121
   at Tomboy.Sync.SyncManager.SynchronizationThread() in C:\Projects\tomboy-gnom
e\Tomboy\Synchronization\SyncManager.cs:line 441

C:\Users\David\AppData\Local\Tomboy>

---

Imported from Launchpad using lp2gh.
- date created: 2012-09-08T16:33:44Z
- owner: dhs-g
- the launchpad url was https://bugs.launchpad.net/bugs/1047912

**Comment by obilodeau** [comment]
(by j-4)
You modified one note, can you attach me the *.note file of this particular note?
(You can exchange unwanted stuff with random letters, but please leave the xml tags)

**Comment by obilodeau** [comment]
(by skrumbholz)
I'm also affected by this issue. Here is my ~/.config/tomboy/tomboy.log

02.10.2012 15:01:49 [ERROR]: Synchronization failed with the following exception: a name did not start with a legal character 32 ( )  Line 4, position 71.
  at Mono.Xml2.XmlTextReader.ReadName (System.String& prefix, System.String& localName) [0x00000] in <filename unknown>:0 
  at Mono.Xml2.XmlTextReader.ReadName () [0x00000] in <filename unknown>:0 
  at Mono.Xml2.XmlTextReader.ReadEntityReference (Boolean ignoreEntityReferences) [0x00000] in <filename unknown>:0 
  at Mono.Xml2.XmlTextReader.ReadReference (Boolean ignoreEntityReferences) [0x00000] in <filename unknown>:0 
  at Mono.Xml2.XmlTextReader.ReadText (Boolean notWhitespace) [0x00000] in <filename unknown>:0 
  at Mono.Xml2.XmlTextReader.ReadContent () [0x00000] in <filename unknown>:0 
  at Mono.Xml2.XmlTextReader.Read () [0x00000] in <filename unknown>:0 
  at System.Xml.XmlTextReader.Read () [0x00000] in <filename unknown>:0 
  at Tomboy.Sync.NoteUpdate..ctor (System.String xmlContent, System.String title, System.String uuid, Int32 latestRevision) [0x00000] in <filename unknown>:0 
  at Tomboy.WebSync.WebSyncServer.GetNoteUpdatesSince (Int32 revision) [0x00000] in <filename unknown>:0 
  at Tomboy.Sync.SyncManager.SynchronizationThread () [0x00000] in <filename unknown>:0 

Where do I find the *.note files on my phone?

**Comment by obilodeau** [comment]
(by skrumbholz)
Strange things happen. :-)

I just deleted all stuff in Tomdroid. Settings - delete all local notes and settings - delete all notes on server. After that I did a backup of my notes in Tomboy/Ubuntu and synced Tomboy/Ubuntu with Ubuntu One. Surprisingly the sync went well now plus the notes were still there! So I did a sync with Tomdroid on my phone. The sync brought back all the notes to my phone. Good.

Then I created a new note with Tomdroid and synced with Ubuntu One. Back to Tomboy/Ubuntu and the sync worked now. The new note made it to my desktop. Great!

I hope this helps.
